lib/rspec_kickstarter/erb_templates.rb

```diff
@@ -26,7 +26,7 @@ module ERBTemplates
     BASIC_NEW_SPEC_TEMPLATE = <<SPEC
 # -*- encoding: utf-8 -*-
 
-require 'spec_helper'
+require 'rails_helper'
```

Comment:
`BASIC_NEW_SPEC_TEMPLATE` is not a template for Rails apps. This part should not be changed.